detectors/aws/ecs/ecs.go

```diff
@@ -263,3 +259,13 @@ func (ecsUtils ecsDetectorUtils) getContainerName() (string, error) {
 	}
 	return hostName, nil
 }
+
+func getCgroupContainerID(fileData []byte) (string, error) {
```

Comment:
Since the error returned here is always `nil`, I think this error might be removable.